t/lib/t/MusicBrainz/Server/Data/Role/GIDEntityCache.pm

```diff
@@ -130,22 +142,20 @@ is ( $test->c->cache->_orig->set_called, 2 );
 };
 
 test 'Cache is transactional (MBS-7241)' => sub {
```

Comment:
I think MBS-7241 makes sense because it concerns issues that occur when the cache is not transactional, whereas MBS-13449 is about performance issues with the implementation of MBS-7241.